### ISSUE TITLE
Documentation not clear how to configure as a helper for grunt-contrib-jasmine

### ISSUE THREAD

**Issue by jeremyhill-up** [Updater/jasmine-fail-fast#6]
I'm trying to get this into my grunt-contrib-jasmine config and I'm struggling to set it up as a helper.  The documentation reads:

> grunt-contrib-jasmine
> Set up as a helper, optionally overriding the default helpers path.

I tried both listing the jasmine-fail-fast.js file directly as a helper in my Grunt config:

```
},
jasmine: {
  default: {
    src: ['<%= dirs.dest %>/<%= app.filename %>.js'],
    options: {
      summary: true,
      specs: 'specs/javascripts/*.js',
      ...
      helpers: [
        'node_modules/jasmine-fail-fast/dist/jasmine-fail-fast.js',
        ...
      ],
      vendor: [
      ...
      ],
      junit: {
        path: 'target/test-results/',
        consolidate: false
      }
    }
  }
},
```

 as well as making a helper file that contained the snippet:

```
var failFast = require('jasmine-fail-fast');
jasmine.getEnv().addReporter(failFast.init());

```

I get ReferenceErrors either way, about 'require' or 'exports' not being defined, ostensibly because grunt-contrib-jasmine aren't running the helpers in a node context or whatever.  Sorry, not sure what terminology to use :)

I'm using "grunt-contrib-jasmine": 1.0.1 which I believe runs Jasmine 2.2.0 by default.

Anyway, what am I missing in the docs?

**Comment by pmowrer** [comment]
I agree the docs are vague, but it sounds like you're doing everything right in using a helper file (which is what the docs are trying to convey, poorly). Where did you place the helper file? What errors did you receive specifically with the helper file?

**Comment by jeremyhill-up** [comment]
Since we're using grunt and grunt-contrib-jasmine, I've configured a helper in the 'helpers' section of my jasmine task, (as seen above) identical to the 'Basic Usage' of the grunt-contrib-jasmine docs.

Where I placed the helper file didn't seem particularly important given it's a string to a file of your choosing, but I used:

```
  helpers: [
            'specs/javascripts/helpers/fail-fast.js',
```

Which had contents: 

```
var failFast = require('jasmine-fail-fast');
jasmine.getEnv().addReporter(failFast.init());
```

And the relevant grunt output is:

```
C:\<snip>>grunt
Running "build:dev" task

Running "setenv:dev" (setenv) task
...

Running "jshint:console" (jshint) task
>> 1 file lint free.

Running "jasmine:default" (jasmine) task
Testing jasmine specs via PhantomJS


>> ReferenceError: Can't find variable: require at
>> ..\..\..\..\..\..\..\C:\<snip>\specs\javascripts\helpers\fail-fast.js:2 global code
```

Does that help at all?

**Comment by pmowrer** [comment]
It's a bit difficult to say what's at fault here, but I'd guess there's more to getting this to work with `grunt-contrib-jasmine` than perhaps we know. I've never tried it and haven't used grunt in years so I'm afraid I'm not great help. 

Do you think this SO answer may lead in the right direction (see linked post)?
http://stackoverflow.com/a/34245789/89594

Or perhaps this one?
http://stackoverflow.com/a/34246531/89594

**Comment by jeremyhill-up** [comment]
Ah, well, we use grunt-karma to run tests in IE11 on our Windows build slave, grunt-contrib-jasmine elsewhere, so the karma/jasmine config mentioned wouldn't apply.

FWIW, I ended up just copying the entire jasmine-fail-fast.js file and stripping out requirejs/node module stuff.  I'm afraid I don't know what the obj.__esModule, and _interopRequireDefault stuff is for, so I removed it and relied on just using the same lodash version as the rest of the project.  I'm sure it was pretty stupid, but it'll do for now.

Thanks for your help, I guess I can close this out.
